features/base/test/test_nis.py

```diff
@@ -0,0 +1,34 @@
+from helper.utils import execute_remote_command as run
+from helper.utils import read_file_remote
+import pytest
+
+
+@pytest.mark.security_id(171)
+def test_for_nis(client, non_container):
+    """
+    Check if we have no NIS related entries.
+    A passwd will then have an entry like this:
+    +::::::
+
+    Also the /etc/nsswitch will have entries as well.
+    """
+
+    for nssl_file in ["passwd", "shadow", "group", "gshadow"]:
+        #  Name Service Switch libraries
+        nssl = run(client, cmd=f"getent {nssl_file}")
+        # Assume that when no + is prsent that the split will never return more
+        # than one element.
+        assert 1 == len(nssl.split("+")), f"NIS entry detected in {nssl_file}!"
```

Comment:
```suggestion
        assert not any(line.strip() == '+' for line in nssl), f"NIS entry detected in {nssl_file}!"
```

Otherwise you test for the + character in comments, usernames and such. Please re-read the rule or explain it to me :)